src/harness/testcases/WINNF_FT_S_BPR_testcase.py

```diff
@@ -0,0 +1,312 @@
+
+    for i, index in enumerate(
+        index_of_devices_with_success_registration_response):
+      if 'installationParam' in config['registrationRequests'][index]:
+        cbsd_information = config['registrationRequests'][index]
+      else:
+        for conditional in config['conditionalRegistrationData']:
+          if (config['registrationRequests'][index]['fccId'] ==
+              conditional['fccId']) and (
+                  config['registrationRequests'][index]['cbsdSerialNumber'] ==
+                  conditional['cbsdSerialNumber']):
+            cbsd_information = conditional
+      logging.info('Looking at Grant response number: %d', i)
+      cbsd_lat = cbsd_information['installationParam']['latitude']
+      cbsd_lon = cbsd_information['installationParam']['longitude']
+      cbsd_ant_azi = cbsd_information['installationParam']['antennaAzimuth']
+      cbsd_ant_beamwidth = cbsd_information['installationParam'][
+          'antennaBeamwidth']
+      # Check if CBSD is in the Border Sharing Zone, Get the closest point.
+      (is_in_sharing_zone, closest_point_lat,
+       closest_point_lon) = utils.CheckCbsdInBorderSharingZone(
+           cbsd_lat, cbsd_lon, cbsd_ant_azi, cbsd_ant_beamwidth)
+      logging.info('CBSD is in Border Sharing Zone?: %s', is_in_sharing_zone)
+      if not is_in_sharing_zone:
+        continue  # CBSD not in the sharing zone; no PFD check is required.
+
+      # Proceed with PFD calculation
+      logging.info('Closest point in the border: Lat is %f', closest_point_lat)
+      logging.info('Closest point in the border: Long is %f', closest_point_lon)
+      requested_eirp = config['grantRequests'][i]['operationParam']['maxEirp']
+      # Calculate PL
+      cbsd_height = cbsd_information['installationParam']['height']
+      is_cbsd_indoor = cbsd_information['installationParam']['indoorDeployment']
+      closest_point_height = 1.5  # According to the spec
+      freq_mhz = 3625.  # Always in all SAS
+      propagation = wf_itm.CalcItmPropagationLoss(
+          cbsd_lat,
+          cbsd_lon,
+          cbsd_height,
+          closest_point_lat,
+          closest_point_lon,
+          closest_point_height,
+          cbsd_indoor=is_cbsd_indoor,
+          freq_mhz=freq_mhz)
+      pl = propagation.db_loss
+      logging.info('Propagation - db_loss: %f', propagation.db_loss)
+      bearing = propagation.incidence_angles.hor_cbsd
+      logging.info('Bearing: %f', propagation.incidence_angles.hor_cbsd)
+      # Calculate effective antenna gain
+      max_ant_gain = cbsd_information['installationParam']['antennaGain']
+      effective_ant_gain = antenna.GetStandardAntennaGains(
+          bearing, cbsd_ant_azi, cbsd_ant_beamwidth,
+          max_ant_gain)
+      logging.info('Effective Antenna Gain: %f', effective_ant_gain)
+      # Arrive at 'P'
+      p = requested_eirp - max_ant_gain + effective_ant_gain
+      # Calculate PFD
+      pfd = p - pl + 32.6
```

Comment:
I believe the math of above three lines are correct. However, if making them more inline with the equation 8.13.1 in the test spec. it will be more straightforward to follow.

Reply:
Fixed. Changed it to look the same as Equation 8.13.1 in the spec.